semaphore/integration_tests.py

```diff
@@ -0,0 +1,128 @@
+#/usr/bin/env python
+from datetime import date, time, datetime, timedelta
+from random import randrange, getrandbits
+import json
+import requests
+
+SPLUNK_HOST = "https://mbta.splunkcloud.com:8089"
+SAVED_SEARCH = "Top 10 trip plans %2B 10 latest trip plans"
+OTP_DEV = "https://dev.otp.mbtace.com"
+OTP_PROD = "https://prod.otp.mbtace.com"
+
+PREDEFINED_PLANS = [
+    # Airport shuttles 1
+    {"Terminal E - Arrivals Level::42.369344,-71.020238", "Wood Island::42.37964,-71.022865"},
+
+    # Airport shuttles 2
+    {"Assembly::42.392811,-71.077257", "Terminal C - Departures Level::42.366635,-71.017167"},
+
+    # Connection between bus routes in the middle of the routes
+    {"Lexington St @ Willow St::42.4732,-71.17253", "Main St, Woburn, MA, USA::42.4928705,-71.1544787"},
+
+    # Logan express
+    {"Back Bay::42.34735,-71.075727", "Logan International Airport, Boston, MA, USA::42.3658907,-71.017547"}
+]
+
+
+def splunk_json_request(path):
+    return requests.get(f"{SPLUNK_HOST}{path}",
+                        params={"output_mode": "json"},
+                        auth=("lboyarsky@mbta.com", "XXX"),
```

Comment:
if you can't get the tokens working, can you then have Ian create a dedicated Splunk account instead of using your own?

Reply:
It's not that I cannot get them working, it's just that I don't have access to create them so I'm waiting for @ianwestcott to do it.